Sources/ArrayExtensions.swift

```diff
@@ -196,6 +196,14 @@ extension Array where Element: Equatable {
     public func unique() -> Array {
         return reduce([]) { $0.contains($1) ? $0 : $0 + [$1] }
     }
+    
+    /// EZSE: Break array into arrays of desired step (inspired by ruby's `step`)
+    public func step(_ stepSize: Int) -> [[Element]]? {
```

Comment:
Ah its to break an array into smaller arrays of specified size.

Use case: I have a view with 2 columns of check boxes, and I am populating it from an array returned from the backend. The array does not have a fixed size, so I have to load xib into the view.

I have to go row by row and add 2 checkboxes to each row, before moving to the next 1, so I had the need to break it up into arrays of smaller chunk